### ISSUE TITLE
Create tickets for accessibility violations

### ISSUE THREAD

**Issue by JoeTice** [department-of-veterans-affairs/va.gov-team#71751]
### Description
Two issues were surfaced in exploring the causes of accessibility alerts that were happening after updating to a newer version of Axe

### Tasks
- [x] Draft tickets using a description/tasks/acceptance criteria format
- [ ] Review w/ Joe T and Peter

### Acceptance Criteria
- [ ] Tickets are created and assigned to the right people
